MonoGame.Framework/Input/MouseCursor.SDL.cs

```diff
@@ -34,21 +34,29 @@ private static void PlatformInitalize()
 
         private static MouseCursor PlatformFromTexture2D(Texture2D texture, int originx, int originy)
         {
-            IntPtr handle;
+            if (texture.Format != SurfaceFormat.Color && texture.Format != SurfaceFormat.ColorSRgb)
```

Comment:
Wow.  DX doesn't implement `Mouse.PlatformSetCursor` or `MouseCursor.PlatformFromTexture2D`.  I've moved the texture format check to `MouseCursor.FromTexture2D` and I'll see about implementing the DX side in a separate PR.